.azure/pipeline.yaml

```
@@ -0,0 +1,114 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - main
+
+pr:
+  branches:
+    include:
+      - "*"
+
```

Add the shared resources needed to use the shared templates

```suggestion
resources:
  repositories:
    - repository: pipeline-templates
      type: github
      name: ogcio/building-blocks-pipelines
      ref: main
      endpoint: ogcio

```